content/04-postgresql/11-reading-and-querying-data/03-joining-tables.mdx

```diff
@@ -359,3 +359,74 @@ Joins are an essential part of what makes relational databases powerful and flex
 Prisma allows you to [define relations](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/relations) between models in the Prisma schema file.  You can then use [relation queries](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/relation-queries) to work with data that spans multiple models.
 
 </PrismaOutlinks>
+
+## FAQ
+
+<details><summary>Does PostgreSQL support outer joins?</summary>
+
+Yes, PostgreSQL supports outer joins. For example, you can use `LEFT OUTER JOIN` or just `LEFT JOIN` like in the following:  
+
+```sql
+SELECT
+    *
+FROM
+    table_1
+LEFT JOIN table_2
+    ON table_1.id = table_2.table_1_id;
+```
+
+</details>
+
+<details><summary>What is a Lateral Join in PostgreSQL?</summary> 
+
+Yes, the `LATERAL` key word in [PostgreSQL](https://www.postgresql.org/docs/9.3/sql-select.html#SQL-FROM) can precede a sub-SELECT FROM item and allows the sub-SELECT to refer to columns of FROM items that appear before it in the FROM list. (Without LATERAL, each sub-SELECT is evaluated independently and so cannot cross-reference any other FROM item.)
```

Comment:
The "Yes" here doesn't really match the section question.

Reply:
I've also been trying to link to PostgreSQL docs at their "current" version instead of the 9.3 that shows up in all of the Google results.  To do so, you just have to click "Current" in the "Supported Versions" selector at the top of their docs pages and it'll substitute the word "current" in the URL which redirects to the docs for their latest release instead of a specific version.